### PR TITLE
Add command line support for thermal control 

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
**- What I did**
1. Add a "show platform fanstatus" command to allow user fetch FAN status data.
2. Add a "show platform temperature" command to allow user fetch thermal status data.

**- How I did it**
1. Add a fanshow script to get FAN status from database and display it in command line
2. Add a tempershow script to get thermal status from database and display it in command line
3. Add two sub-commands under "platform" group

**- How to verify it**
1. Issue "show platform fanstatus" in command line and see the output.
2. Issue "show platform temperature" in command line and see the output.

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -1469,6 +1469,20 @@ def ssdhealth(device, verbose, vendor):
     options += " -e" if vendor else ""
     run_command(cmd + options, display_cmd=verbose)
 
+# 'fanstatus' subcommand ("show platform fanstatus")
+@platform.command()
+def fanstatus():
```
Comment (jleveque) [inline show/main.py]: I understand we currently have `show platform psustatus`, but I wonder if we should shorten this command to simply, `show platform fans` or `show platform fan`, and in the future, shorten the other command to `show platform psu`.

File: scripts/tempershow [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            print('No tempeature data available\n')
+
+
+if __name__ == "__main__":
+    temperShow = TemperShow()
+    temperShow.show()
+    
+
+    
+
```
Comment (jleveque) [inline scripts/tempershow]: Delete trailing empty lines

**Comment by jleveque** [comment]
@Junchao-Mellanox: Please fix the new conflict.

**Comment by liat-grozovik** [comment]
@stepanblyschak please review as well

**Pull request merged**